lib/plurimath/asciimath/constants.rb

```diff
@@ -0,0 +1,315 @@
+        equiv
+        Sigma
+        Omega
+        omega
+        aleph
+        infty
+        sigma
+        frown
+        notin
+        angle
+        succ
+        prec
+        cong
+        \\\\
+        star
+        odot
+        cdot
+        rarr
+        darr
+        prop
+        lArr
+        rArr
+        uarr
+        beta
+        hArr
+        harr
+        larr
+        grad
+        circ
+        >->>
+        zeta
+        oint
+        iota
+        /_\\
+        sube
+        sup
+        sub
+        !in
+        int
+        eta
+        top
+        |><|
+        __|
+        |__
+        ":'"
+        Phi
+        supe
+        Psi
+        psi
+        chi
+        >-=
+        -<=
+        mgt
+        mlt
+        ><|
+        |==
+        |--
+        vvv
+        vee
+        nnn
+        cap
+        ^^^
+        div
+        ast
+        bot
+        del
+        and
+        neg
+        not
+        uuu
+        cup
+        iff
+        phi
+        rho
+        tau
+        |->
+        >->
+        <=>
+        ->>
+        ^^
+        nn
+        vv
+        TT
+        EE
+        ox
+        xx
+        o+
+        o.
+        |><
+        _|_
+        ***
+        **
+        -:
+        ~~
+        //
+        pm
+        O/
+        ->
+        =>
+        +-
+        to
+        Xi
+        AA
+        if
+        uu
+        or
+        lt
+        ne
+        nu
+        mu
+        ZZ
+        RR
+        QQ
+        NN
+        CC
+        >>
+        <<
+        ~|
+        !=
+        >-
+        -<
+        ~=
+        -=
+        pi
+        xi
+        oo
+        Pi
+        :.
+        gg
+        ll
+        ge
+        >=
+        le
+        <=
+        gt
+        |~
+        in
+        /_
+        >
+        @
+        /
+        *
+        -
+        <
+        =
+        +
+      ].freeze
+      UNARY_CLASSES = %i[
+        mathfrak
+        mathcal
+        arccos
+        arcsin
+        arctan
+        mathsf
+        mathbb
+        mathbf
+        mathtt
+        ubrace
+        obrace
+        cancel
+        tilde
+        floor
+        ceil
+        ddot
+        coth
+        csch
+        sech
+        sinh
+        tanh
+        cosh
+        sqrt
+        norm
+        text
+        sec
+        sin
+        tan
+        cos
+        exp
+        gcd
+        glb
+        lcm
+        lub
+        cot
+        csc
+        det
+        dim
+        max
+        min
+        abs
+        bar
+        dot
+        hat
+        vec
+        ul
+        ln
+        f
+        g
+      ].freeze
+      BINARY_CLASSES = %i[
+        underset
+        stackrel
+        overset
+        color
+        prod
+        frac
+        root
+        sum
+        mod
+        log
+      ].freeze
+      RPAREN = %i[
+        )
+        }
+        :)
+        \]
+        :}
+      ].freeze
+      LPAREN = %i[
+        (
+        {
+        (:
+        \[
+        {:
+      ].freeze
```

Comment:
Can we use `PARENTHESIS.keys` and `PARENTHESIS.values` instead of `LPAREN` and `RPAREN`?